src/InputToACObject.cs

```diff
@@ -0,0 +1,336 @@
+                    {
+                        continue;
+                    }
+                    if (configureExit)                                                                      // wenn True -> kein Überbereich(configure voip // configure network) in dem die Konfig definiert wird
+                    {
+                        getConfigureIdent(line, out configureExit, out ident);
+                        continue;
+                    }
+                    if (subIdentExit)
+                    {
+                        getIdentNameAndValue(line, out configureExit, out subIdentExit, out subIdent, out subIdentValue);                   // wenn true -> keien Bereich(networkDev / interfaceNetworkIf) in dem Konfig konfiguriert wird
+                        var Name = returnRealName(subIdent);
+                        if (Name == null)
+                        {
+                            continue;
+                        }
+                        if (subIdent == ParserVariables.networkDev)
+                        {
+                            Networkdev newListNetworkDev = new Networkdev();
+                            networkDevs.Add(newListNetworkDev);                                                                           //neue Liste wird erstellt für jede Bereich von networkDev
+                            AC = ListParsing(AC, Name, subIdentValue, networkDevListTndex, AC.configureNetwork.networkdev, subIdent);
+                        }
+                        else if (subIdent == ParserVariables.interfaceNetwokIf)
+                        {
+                            Interfacenetworkif newListInterfaceNetworkIf = new Interfacenetworkif();
+                            interfaceNetworkIfs.Add(newListInterfaceNetworkIf);
+                            AC = ListParsing(AC, Name, subIdentValue, interfaceNetwokIfListTndex, AC.configureNetwork.interfacenetworkif, subIdent);
+                        }
+                        else if (subIdent == ParserVariables.proxySet)
+                        {
+                            Proxyset newListProxySet = new Proxyset();
+                            proxySets.Add(newListProxySet);
+                            AC = ListParsing(AC, Name, subIdentValue, proxySetListTndex, AC.configureviop.proxyset, subIdent);
+                        }
+                        else if (subIdent == ParserVariables.proxyIp)
+                        {
+                            Proxyip newListProxyIp = new Proxyip();
+                            proxyIps.Add(newListProxyIp);
+                            AC = ListParsing(AC, Name, subIdentValue, proxyIpListTndex, AC.configureviop.proxyip, subIdent);
+                        }
+                        continue;
+                    }
+
+                    if (ident == "configure network" && subIdent == ParserVariables.networkDev)
+                    {
+                        var Name = ParserGrammar.NameParser.Parse(line);
```

Comment:
put this in extra methods. This is to hard to read. Please to it for all else if bodys to